packages/astro/test/redirects.test.js

```diff
@@ -217,7 +245,7 @@ describe('Astro.redirect', () => {
 		});
 	});
 
-	describe('config.build.redirects = false', () => {
+	describe.skip('config.build.redirects = false', () => {
```

Comment:
Is it intended? If so, could you add a comment or `TODO` to tell us when we should unskip it?

Reply:
good catch, this was just for testing locally, unskipped